app/graphql/mutations/users/update_user_details_mutation.rb

```diff
@@ -0,0 +1,29 @@
+module Mutations
+  module Users
+    class UpdateUserDetailsMutation < Mutations::BaseMutation
+      description 'Updates a User with the access_token'
+      null true
+      argument :personalAccessToken, String, required: false
```

Comment:
On the ruby side, these will actually be the `snake_case` versions for these args (i.e. `personal_access_token`, etc.)

Reply:
For reference, if you `snake_case` in ruby, the mutation automatically gets converted to `camelCase`. As makes sense (in retrospect), just snake_case **everywhere** and camelCase will happen everywhere in JS

Thanks graphql-ruby 👍 !